### PR TITLE
PMM-5728 management add external

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

File: api/managementpb/external.proto [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  // Node identifier on which a external service is been running.
+  string node_id = 8;
+  // Environment name.
+  string environment = 9;
+  // Cluster name.
+  string cluster = 10;
+  // Replication set name.
+  string replication_set = 11;
+  // Custom user-assigned labels for Service.
+  map<string, string> custom_labels = 15;
```
Comment (askomorokhov) [inline api/managementpb/external.proto:48]: Can it be 12?

Comment (BupycHuk) [reply api/managementpb/external.proto:48]: yes, but I'm pretty sure that we will add extra fields later.

**Comment by askomorokhov** [comment]
and CI fails :)

**Comment by AlekSi** [comment]
https://github.com/percona/pmm/pull/523 should be merged into this branch. And yes, CI fails.

**Comment by AlekSi** [comment]
Please check comments at https://github.com/percona/pmm/pull/523

**Comment by AlekSi** [comment]
There are conflicts now

**Comment by AlekSi** [comment]
@BupycHuk please merge when time is right

**Pull request merged**